main.py

```diff
@@ -214,6 +228,21 @@ async def take_game_decision(adb_instance: ADB, config: AluneConfig):
 
     await buy_from_shop(adb_instance, config)
 
+    white_orbs_to_collect = screen.get_button_on_screen(screenshot, Button.white_orb)
```

Comment:
There's a way to get all matches of cv2 match template. This solution is fine for now, but looping all matches would save time.